### PR TITLE
平行四界Quadimension Discs

### PR DESCRIPTION
**Pull request by Yesterday17**
- [x] 平行四界Quadimension X-1
- [x] 平行四界Quadimension X-2

### CODE REVIEW

File: album/QMEP-0002.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[album]
+title = "南北极星 Vol.1"
+artist = "洛天依、乐正绫、星尘"
+date = 2016-11-30
+type = "normal"
+catalog = "QMEP-0002"
+tags = ["洛天依", "乐正绫", "星尘", "平行四界 EP"]
```
Comment (tuzi3040) [inline album/QMEP-0002.toml:7]:
```suggestion
tags = ["洛天依", "乐正绫", "星尘", "平行四界", "EP"]
```
May be better to add a single independent tag named `EP`, and also `album` and so on.

File: tag/[AAAA] 平行四界Quadimension.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[[tag]]
+name = "平行四界Quadimension"
+includes = [
+  "星尘",
+  "海伊",
+  "苍穹",
+  "赤羽",
+  "诗岸",
+  "平行四界 EP",
```
Comment (tuzi3040) [inline tag/[AAAA] 平行四界Quadimension.toml:9]:
```suggestion
  "平行四界",
```

**Review by tuzi3040 — CHANGES_REQUESTED** [review]

File: album/QMEP-0002.toml (hunk 1 above)
Comment (Yesterday17) [reply album/QMEP-0002.toml:7]: Making `EP`/`Single`/`Album` a single category might be strange. Something like `平行四界:default` might be better for those albums(without series).

Comment (tuzi3040) [reply album/QMEP-0002.toml:7]: Or `平行四界:EP`.

Comment (Yesterday17) [reply album/QMEP-0002.toml:7]: `平行四界:EP` Seems good.

**Review by tuzi3040 — APPROVED** [review]

**Pull request merged**